RELEASE_NOTES.md

```diff
@@ -1,5 +1,6 @@
 ##### 0.13.1
 
+* Remove support for multiple overriding configs
```

Comment:
Worth bumping the version for this with 0.13.1 already published

Reply:
Whoops, I meant to release all these breaking changes together; I thought not putting the date would make it not trigger a release.